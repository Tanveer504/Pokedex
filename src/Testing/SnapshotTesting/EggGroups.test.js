import renderer from 'react-test-renderer';
import EggGroup from '../../components/PokemonDetails/EggGroup';

it('should have the right EggGroup component snapshot', () => {
    const tree = renderer.create(<EggGroup />).toJSON();
    expect(tree).toMatchSnapshot();
});