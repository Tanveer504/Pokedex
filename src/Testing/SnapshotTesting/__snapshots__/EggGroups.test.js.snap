// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have the right Header component snapshot 1`] = `
<div
  className="header"
>
  <h1>
    Pokédex
  </h1>
  <h2>
    Search for any Pokémon that exists on the planet
  </h2>
</div>
`;
